sponsor.html

```diff
@@ -0,0 +1,404 @@
+                <td class="align-middle"><div class="benefit recruit"></div></td>
+              </tr>
+              <tr>
+                <td>Pre-event email to participants</td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td class="align-middle"><div class="benefit recruit"></div></td>
+              </tr>
+              <tr>
+                <th scope="colgroup">Branding</th>
+              </tr>
+              <tr>
+                <td>Logo on Website & T-shirt</td>
+                <td class="align-middle"><div class="benefit brand"></div></td>
+                <td class="align-middle"><div class="benefit brand"></div></td>
+                <td class="align-middle"><div class="benefit brand"></div></td>
+                <td class="align-middle"><div class="benefit brand"></div></td>
+                <td class="align-middle"><div class="benefit brand"></div></td>
+                <td class="align-middle"><div class="benefit brand"></div></td>
+              </tr>
+              <tr>
+                <td>Distribution of marketing materials</td>
+                <td class="align-middle"><div class="benefit brand"></div></td>
+                <td class="align-middle"><div class="benefit brand"></div></td>
+                <td class="align-middle"><div class="benefit brand"></div></td>
+                <td class="align-middle"><div class="benefit brand"></div></td>
+                <td class="align-middle"><div class="benefit brand"></div></td>
+                <td class="align-middle"><div class="benefit brand"></div></td>
+              </tr>
+              <tr>
+                <td>Opportunity to speak at the opening/closing ceremony</td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td class="align-middle"><div class="benefit brand"></div></td>
+                <td class="align-middle"><div class="benefit brand"></div></td>
+                <td class="align-middle"><div class="benefit brand"></div></td>
+              </tr>
+              <tr>
+                <td>Social Media Shoutout</td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td class="align-middle"><div class="benefit brand"></div></td>
+                <td class="align-middle"><div class="benefit brand"></div></td>
+                <td class="align-middle"><div class="benefit brand"></div></td>
+              </tr>
+              <tr>
+                <td>Branded Mini-Challenge</td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td class="align-middle"><div class="benefit brand"></div></td>
+                <td class="align-middle"><div class="benefit brand"></div></td>
+              </tr>
+              <tr>
+                <td>"Co-hosted by..."</td>
+                <td></td>
```

Comment:
Should these dots be colored like in the sponsorship prospectus? I see there they are different colors

Reply:
yeah rn theyre just all black, im gonna add the colors in later